src/pybindings/pycontroller/TestDeviceController.py

```diff
@@ -0,0 +1,119 @@
+from pybindings.PyChip.chip import Controller, PersistentStorageDelegate, Platform, RendezvousParameters
+from pybindings.PyChip.chip import SetupPayload, Transport, DeviceLayer
+from pybindings.PyChip.chip import QRCodeSetupPayloadParser
+from pybindings.PyChip import ChipExceptions
+
+import time
+import threading
+import ctypes
+import atexit
+import logging
+from sys import platform
+
+is_linux_platform = platform == "linux" or platform == "linux2"
+logger = logging.getLogger("ChipBLEMgr")
+logger.setLevel(logging.DEBUG)
+
+
+ChipExceptions.CHIPErrorToException(Platform.MemoryInit())
+platform_manager = DeviceLayer.PlatformMgr()
+
+# This has to go before InitChipStack or else there will be GLib critical errors.
+if is_linux_platform:
+    DeviceLayer.Internal.BLEMgrImpl().ConfigureBle(0, True)  # Central is set to True
```

Comment:
I find this path worrying in terms of a python API: requiring MemoryInit and BLEMgrImpl::ConfigureBle and similar. This is extreamly low level and feels more like 'script C++ code' than python automation.

Reply:
This was the idea, but the more I worked on it, doing this it ended up being a nearly a 1:1 mapping of what we were wrapping. For example, for this we still need to configure BLE on linux, so we can make a C++ function to call this and then have python call this C++ function, which is pretty redundant. Also, please note that the previous implementation did  this [here](https://github.com/project-chip/connectedhomeip/blob/630fd1de0ab5c2857b335924aa7c110831bb46db/src/controller/python/chip/native/StackInit.cpp#L45). If we do the alternative, we will find ourselves writing python modules in C++, when we can write python modules in python. Also, this allows for quicker development of anything in python. We would have to compile for any changes done in the wrapper, bind the wrapper, then implement it in python and then run it. This way, you can immediately run the changes to the controller. Can you please elaborate more on what is worrisome if the previous implementation did this, with extra layers and no objects?